WordPress/Classes/ViewRelated/Blog/Blog Dashboard/DashboardCard.swift

```diff
@@ -84,6 +84,17 @@ enum DashboardCard: String, CaseIterable {
         }
     }
 
+    /// Specifies whether the card settings should be applied across
+    /// different sites or only to a particular site.
+    var settingsType: SettingsType {
```

Comment:
I think the default value is fine since there's around 12 cases that we would need to list.